daphne_worker/src/config.rs

```diff
@@ -486,6 +492,7 @@ impl<'srv> DaphneWorkerRequestState<'srv> {
 
     pub(crate) fn dap_abort_to_worker_response(&self, e: DapAbort) -> Result<Response> {
         let status = if matches!(e, DapAbort::Internal(..)) {
+            self.error_reporter.report_abort(&e);
```

Comment:
Note we already log the error via `error!()` below, so this may end up being redundant, at least for the default error reporter. Maybe the default behavior should be a no-op?